prussia_rt/src/exceptions/v_common/trap.rs

```diff
@@ -0,0 +1,26 @@
+use prussia_debug::println_ee;
+
+use crate::{cop0::CoP0Dump, thread::ThreadControlBlock};
+
+/// Trigger Trap exception.
+pub unsafe fn trigger_trap_exception() {
+    println_ee!("Triggering trap.");
+
+    core::arch::asm!("
+        li $t1, 0x5
+        li $t2, 0x5
+        teq $t1, $t2
```

Comment:
You can probably make this more succinct by trapping if `$zero` is equal to `$zero`.

Reply:
Fixed accordingly.